vcpkg.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "makou-reactor",
   "version": "2.0.0",
```

Comment:
This should have been bumped to `2.1.0`

Reply:
My bad, you're right, good catch.

Reply:
Created PR: https://github.com/myst6re/makoureactor/pull/211